frontend/src/components/TournamentSchedule.js

```diff
@@ -294,7 +294,7 @@ const TournamentSchedule = () => {
                           ]
                         )}
                       >
-                        <TournamentMatch
+                        <MatchCard
```

Comment:
need to add useUCRegistrations prop here?

Reply:
added